torchvision/ops/decode_png.py

```diff
@@ -0,0 +1,40 @@
+import torch
```

Comment:
I was wondering if those files should be moved to `torchvision/io/image.py` instead.
They are decoding operators, so not exactly IO operators, but we do have some video decoding there.

Thoughts?

Reply:
Hi, 
It makes sense. My initial understanding was that ops was for `python` to `C++` bindings. I did not go through the whole codebase. I put that in there for that reason. 

I'll move the code